src/fish.cpp

```diff
@@ -57,6 +57,14 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA
 #define PATH_MAX 4096
 #endif
 
+// container to hold the options specified within the command line
+class command_line_switches_t {
+public:
+    std::vector<std::string> cmds;              // commands to be executed in place of interactive shell
```

Comment:
heh, no problem; I mostly just lifted the variable name and added the new one, so better names are appreciated :-)